Configuration/AlCa/python/autoCond.py

```diff
@@ -27,24 +27,18 @@
     'run2_data'                    : '124X_dataRun2_v2',
     # GlobalTag for Run2 data 2018B relvals only: HEM-15-16 fail
     'run2_data_HEfail'             : '124X_dataRun2_HEfail_v2',
-    # GlobalTag for Run2 data relvals: allows customization to run with fixed L1 menu
-    'run2_data_relval'             : '124X_dataRun2_relval_v2',
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi'      : '124X_dataRun2_PromptLike_HI_v1',
     # GlobalTag with fixed snapshot time for Run2 HLT RelVals: customizations to run with fixed L1 Menu
-    'run2_hlt_relval'              : '123X_dataRun2_HLT_relval_v3',
-    # GlobalTag for Run3 HLT: identical to the online GT (126X_dataRun3_HLT_v1) but with snapshot at 2023-01-06 18:15:00 (UTC)
-    'run3_hlt'                     : '126X_dataRun3_HLT_frozen_v1',
-    # GlobalTag with fixed snapshot time for Run3 HLT RelVals: customizations to run with fixed L1 Menu
-    'run3_hlt_relval'              : '126X_dataRun3_HLT_relval_v2',
-    # GlobalTag for Run3 data relvals (express GT) - identical to 126X_dataRun3_Express_v1 but with snapshot at 2023-01-06 18:15:00 (UTC)
-    'run3_data_express'            : '126X_dataRun3_Express_frozen_v1',
-    # GlobalTag for Run3 data relvals (prompt GT) - identical to 126X_dataRun3_Prompt_v1 but with snapshot at 2023-01-06 18:15:00 (UTC)
-    'run3_data_prompt'             : '126X_dataRun3_Prompt_frozen_v1',
-    # GlobalTag for Run3 offline data reprocessing - snapshot at 2023-01-06 18:15:00 (UTC)
-    'run3_data'                    : '126X_dataRun3_v1',
-    # GlobalTag for Run3 data relvals: allows customization to run with fixed L1 menu
-    'run3_data_relval'             : '126X_dataRun3_relval_v2',
+    'run2_hlt'                     : '123X_dataRun2_HLT_relval_v3',
```

Comment:
Not a problem for HLT per se, but isn't confusing that now `run2_hlt` points to a "relval" GT ? (I was expecting a frozen, non-relval, HLT GT instead)

If it's easier/clearer , there is no problem keeping `run2_hlt_relval`.

Reply:
it's just that `run2_hlt` is not consumed anywhere as a frozen, non-relval GT, we dont even have a queue for it, so it would have been some extra effort to do something that's then not used anywhere. You guys are the real users of this, so I can change it as you wish. Should I revert to `run2_hlt_relval` then @missirol ?

Reply:
>  Should I revert to run2_hlt_relval then ?

Let's do that, to avoid unnecessary changes. Sorry for the back and forth.